client/verta/verta/operations/monitoring/summaries.py

```diff
@@ -261,6 +287,28 @@ def create(self, name, data_type_cls, monitored_entity):
         proto = self._conn.must_proto_response(response, SummaryProto)
         return Summary(self._conn, self._conf, proto)
 
+    def get_or_create(self, name, data_type_cls, monitored_entity):
+        assert issubclass(data_type_cls, data_types._VertaDataType)
+        query = SummaryQuery(names=[name], monitored_entities=[monitored_entity])
+        retrieved = self.find(query)
+        if retrieved and len(retrieved) > 1:
+            warnings.warn(
+                "Found multiple summaries with name: {}, for monitored entity: {}.".format(
```

Comment:
The client's been leaning towards starting error/warning messages in lowercase, and omitting a trailing period
```suggestion
                "found multiple summaries with name: {}, for monitored entity: {}".format(
```